requirements.txt

```diff
@@ -1,14 +1,2 @@
-################################################################################
-# This requirements file has been automatically generated from `Pipfile` with
-# `pipenv-to-requirements`
-#
-#
-# This has been done to maintain backward compatibility with tools and services
-# that do not support `Pipfile` yet.
-#
-# Do NOT edit it directly, use `pipenv install [-d]` to modify `Pipfile` and
-# `Pipfile.lock` and then regenerate `requirements*.txt`.
-################################################################################
-
-pbr
-typing ; python_version < '3.5'
+pbr==5.4.4
```

Comment:
if we use pbr, it is important to keep requirement.txt not frozen (without the version)

Reply:
I do not think there is a straightforward way to generate the requirements files while leaving the version of any dependency blank.

Is `poetry-dynamic-versioning` the plugin you mentioned? Should we try to replace `pbr` with that plugin?